fili-sql/src/main/resources/moduleConfig.properties

```diff
@@ -0,0 +1,153 @@
+# Copyright 2016 Yahoo Inc.
+# Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+
+#The name of this module (required)
+moduleName = fili-sql
+
+# Resource binder is critical to starting the app
+bard__resource_binder = [SET ME IN APPLICATION CONFIG]
+
+# Redis configuration
+bard__redis_host = localhost
+bard__redis_port = 6379
+# timeout of 5 seconds (in milliseconds)
+bard__redis_timeout_ms = 5000
+# namespace all of the keys stored in Redis, only necessary if you wish to support asynchronous queries
+bard__redis_namespace = [SET ME IN APPLICATION CONFIG]
+
+# The channel on which RedisBroadcastChannel can publish/listen to messages, only necessary if you wish to support
+# asynchronous queries
+bard__redisbroadcastchannel_name = preResponse_notification_channel
+
+# Rate Limiting -- Tuned for current cluster limits.  Increase after new hardware available
+bard__request_limit_global = 70
+bard__request_limit_per_user = 2
+bard__request_limit_ui = 52
+
+# setting for druid query weight limit
+bard__query_weight_limit = 100000
+
+# Default the timeout to 10 minutes, in milliseconds
+bard__druid_request_timeout = 600000
+
```

Comment:
Clean out config that isn't specific to your module.
Add module dependency to fili-core